modules/adb-with-private-link-standard/outputs.tf

```diff
@@ -7,4 +7,9 @@ output "dp_workspace_url" {
   // The workspace URL which is of the format 'adb-{workspaceId}.{random}.azuredatabricks.net'
   // this is not named as DATABRICKS_HOST, because it affect authentication
   value = "https://${azurerm_databricks_workspace.dp_workspace.workspace_url}/"
-}
\ No newline at end of file
+}
+
+output "test_vm_password" {
+  value = azurerm_windows_virtual_machine.testvm.admin_password
```

Comment:
It would be useful to add description to this & other outputs  (and inputs)

Reply:
will do

Reply:
though I'm only fixing what wasn't working